packages/calendar/src/eventDialog.html

```diff
@@ -1,5 +1,5 @@
 <md-dialog aria-label="{{::ctrl.eventData.title + ' Event Dialog'}}">
-    <article id="calendar-event-dialogue" class="initialize-dialogue dialogue-container background-light default-border-radius default-shadow">
+    <article id="calendar-event-dialog" class="initialize-dialog dialog-container background-light default-border-radius default-shadow">
```

Comment:
this change will affect any css stylings points to this id and classes

Reply:
Yes, but I did a search and there was no other references with classnames for the old 'dialogue'. So initialize-dialogue and initialize-dialogue-container are probably not currently used.